src/components/pokemon/Pokemon.jsx

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
+import './Pokemon.css';
+
+function Pokemon({name}) {
+    const [pokemonData, setPokemonData] = useState({});
```

Comment:
Hier zou ik ervoor kiezen om de default waarde undefined te maken, zodat je een duidelijker verschil hebt tussen wat wél en niet ingeladen is
```suggestion
    const [pokemonData, setPokemonData] = useState();
```